### PR TITLE
Set packaging defaults for Jenkins

### PR DESCRIPTION
**Pull request by andrewkroh**
Control the defaults for Jenkins in our repo, but allow them to be
overridden by Jenkins parameters.

### CODE REVIEW

**Comment by andrewkroh** [comment]
This will add back the `-alpha` to the snapshots produced by Jenkins from the master branch.

**Comment by andrewkroh** [comment]
Started a package test: https://beats-ci.elastic.co/job/elastic+beats+pull-request+package/4/

File: dev-tools/jenkins_release.sh [hunk 1]
```diff
@@ -16,4 +16,10 @@ cleanup() {
 }
 trap cleanup EXIT
 
+# This controls the defaults used the Jenkins package job. They can be
+# overridden by setting them in the environement prior to running this script.
+export BEAT_VERSION_QUALIFIER="${VERSION_QUALIFIER:-alpha1}"
```
Comment (ruflin) [inline dev-tools/jenkins_release.sh]: So these values would be adjusted based on the branch?

Comment (andrewkroh) [reply dev-tools/jenkins_release.sh]: Yes, that would be the case. The alternative would be to put it in the JJB config in infra, but we want to get away from that model in order to have more control over our Jenkins jobs from our repo.

**Pull request merged**